api/src/main/java/net/kyori/adventure/audience/EmptyAudience.java

```diff
@@ -30,69 +30,72 @@
 import net.kyori.adventure.inventory.Book;
 import net.kyori.adventure.pointer.Pointer;
 import net.kyori.adventure.text.ComponentLike;
-import org.checkerframework.checker.nullness.qual.NonNull;
-import org.checkerframework.checker.nullness.qual.PolyNull;
+import org.jetbrains.annotations.Contract;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
 
 final class EmptyAudience implements Audience {
   static final EmptyAudience INSTANCE = new EmptyAudience();
 
   @Override
-  public @NonNull <T> Optional<T> get(final @NonNull Pointer<T> pointer) {
+  public @NotNull <T> Optional<T> get(final @NotNull Pointer<T> pointer) {
     return Optional.empty();
   }
 
+  @Contract("_, null -> null; _, !null -> !null")
   @Override
-  public <T> @PolyNull T getOrDefault(final @NonNull Pointer<T> pointer, final @PolyNull T defaultValue) {
+  public <T> @UnknownNullability T getOrDefault(final @NotNull Pointer<T> pointer, final @Nullable T defaultValue) {
```

Comment:
This annotation should be on the method below

Reply:
What?

Reply:
It's the supplier methods that need annotating with the `@UnknownNullability` annotations. These methods are fine as the poly-nullability is covered by the `@Contract` annotations.